net-p2p/bitcoin-core/bitcoin-core-25.1-r1.ebuild

```diff
@@ -144,23 +181,41 @@ src_configure() {
 		--enable-util-tx
 		--${wallet}-util-wallet
 		--disable-util-util
+		--without-seccomp
 		$(use_with libs)
 		$(use_with daemon)
-		$(use_with gui)
+		$(use_with gui gui qt5)
 		$(use_with dbus qtdbus)
 		$(use_with system-leveldb)
 		$(use_with system-libsecp256k1)
 	)
 	econf "${myeconfargs[@]}"
 }
 
+src_test() {
+	emake check
+
+	if use daemon ; then
+		local jobs=$(get_makeopts_jobs unset)
+		[[ "${jobs}" == unset ]] && unset jobs
+		set -- "${PYTHON}" test/functional/test_runner.py --ansi --extended ${jobs+--jobs="${jobs}"} --timeout=0
```

Comment:
Just use edo here.

Reply:
Egads! I had no idea that existed. That's going to be useful in many of my other ebuilds as well.